src/todo/items/todo.ts

```diff
@@ -25,11 +25,36 @@ class Todo extends Item {
     this._lineNextText = val;
   }
 
+  _automaticTodo;
+
+  get automaticTodo () {
+    return this._automaticTodo;
+  }
+
+  set automaticTodo ( auto: boolean ){
+    this._automaticTodo = auto;
+  }
+
+  _nextLineNextText;
+
+  get nextLineNextText () {
+    if ( !_.isUndefined ( this._nextLineNextText ) ) return this._nextLineNextText;
+    return this.textDocument.lineAt(this.line.lineNumber + 1).text;
+  }
+
+  set nextLineNextText ( val ) {
+    this._nextLineNextText = val;
+  }
+
   /* EDIT */
 
   makeEdit () {
```

Comment:
This function makes the edits necessary for transforming the previous line to the next line, it shouldn't be touched.